man/BatchtoolsParam-class.Rd

```diff
@@ -80,53 +81,59 @@ batchtoolsRegistryargs(...)
 	a \code{template} are "sge", "slurm", "lsf", "openlava", and
 	"torque". If the template is not given then a default is
 	selected from the \code{batchtools} package.
-	
+
     \item{registryargs}{\code{list()}} Arguments given to the registry
-        created by \code{BatchtoolsParam} to configure the registry and
-        where it's being stored. The \code{registryargs} can be
-        specified by the function \code{batchtoolsRegistryargs()} which
-        takes the arguments \code{file.dir}, \code{work.dir},
-        \code{packages}, \code{namespaces}, \code{source}, \code{load},
-        \code{make.default}. It's useful to configure these option,
-        especially the \code{file.dir} to a location which is accessible
-        to all the nodes on your job scheduler i.e master and
-        workers. \code{file.dir} uses a default setting to make a
-        registry in your working directory.
+	created by \code{BatchtoolsParam} to configure the registry and
```

Comment:
here and elsewhere you editor has replaced spaced (preferred) with tabs; please undo.